res/themes/bash-dark-green/theme.yaml

```diff
@@ -0,0 +1,401 @@
+---
+display:
+  # Specify the display orientation for this theme: portrait, landscape, reverse_portrait, reverse_landscape
+  DISPLAY_ORIENTATION: reverse_portrait
```

Comment:
can we commit this as portrait by default? This way it would fit most people orientation 👍